docs/hardware/kiosk.md

```diff
@@ -64,6 +64,19 @@ Alternately, you can add a game to the kiosk through a shared game link. On the
 
 Please note that for security purposes, the Kiosk ID is only valid for 10 minutes, after which time you will need to generate a new Kiosk ID. There is no limit to how many games can be added to a kiosk.
 
+## Deleting a game from the Kiosk
+
+If you've added a game to the kiosk, you can also remove that game. When you're focused on a game that you added, you should see a delete button underneath. 
+
+![Delete button highlighted on select game](/static/hardware/kiosk/delete-button-highlighted.png)
+
+Use the joystick to navigate to the button. Press the **A** button to select. This will open a pop-up for confirming the deletion. 
+
+![Delete game dailogimage.png](/static/hardware/kiosk/delete-game-modal.png)
+
+
+Move the joystick left or right to select either option. Cancelling keeps the game list as is. Confirming will remove the game from the list of games. 
```

Comment:
Proposed alternate wording: "Move the joystick left or right to select Cancel or Confirm.  If you Confirm deletion, the game will be removed from the Kiosk."